docs/book/src/reference/jobs.md

```diff
@@ -49,7 +49,7 @@ Prow Periodics:
 * [periodic-cluster-api-e2e-upgrade-v1-0-to-main] `./scripts/ci-e2e.sh`
   * GINKGO_FOCUS: `[clusterctl-Upgrade]`
 * [periodic-cluster-api-e2e-mink8s-main] `./scripts/ci-e2e.sh`
-  * GINKGO_SKIP: `[Conformance] [K8s-Upgrade]`
+  * GINKGO_SKIP: `[Conformance] [K8s-Upgrade] [ClusterClass]`
```

Comment:
nit the `|` is missing

Reply:
Right, I was a bit confused but IPv6 was missing. Added `|` and also added the missing `|[IPv6]`